lib/pages/planning_page/manage_budget_page.dart

```diff
@@ -0,0 +1,162 @@
+          ListView.builder(
+            shrinkWrap: true,
+            itemCount: budgets.length,
+            itemBuilder: (context, index) {
+              return Dismissible(
+                  key: Key(budgets[index].idCategory.toString()),
+                  background: Container(
+                    padding: const EdgeInsets.only(right: 20.0),
+                    alignment: Alignment.centerRight,
+                    color: Colors.red,
+                    child: const Text(
+                      'Delete',
+                      textAlign: TextAlign.right,
+                      style: TextStyle(color: Colors.white),
+                    ),
+                  ),
+                  onDismissed: (direction) {
+                    deleteBudget(budgets[index], index);
+                  },
+                  child: BudgetCategorySelector(
+                    categories: categories,
+                    categoriesAlreadyUsed: categories.where((element) => budgets.map((e) => e.name).contains(element.name)).map((e) => e.name).toList(),
+                    budget: budgets[index],
+                    initSelectedCategory: categories
+                            .where((element) =>
+                                element.id == budgets[index].idCategory)
+                            .isEmpty
+                        ? categories[0]
+                        : categories
+                            .where((element) =>
+                                element.id == budgets[index].idCategory)
+                            .first,
+                    onBudgetChanged: (updatedBudget) {
+                      updateBudget(updatedBudget, index);
+                    },
+                  ));
+            },
+          ),
+          TextButton.icon(
+              icon: Icon(
+                Icons.add_circle,
+                color: Theme.of(context).colorScheme.secondary,
+              ),
+              onPressed: () {
+                setState(() {
+                  budgets.add(Budget(
+                      active: true,
+                      amountLimit: 100,
+                      idCategory: categories[0].id!,
+                      name: categories[0].name));
+                });
+              },
+              label: Text(
+                "Add category budget",
+                style: Theme.of(context)
+                    .textTheme
+                    .titleSmall!
+                    .apply(color: Theme.of(context).colorScheme.secondary),
+              )),
+        ])),
+        const SizedBox(height: 10),
+        Text(
+            "Your monthly budget will be: ${budgets.isEmpty ? 0 : budgets.fold(0, (sum, e) => sum + e.amountLimit.toInt())}€"),
+        const SizedBox(height: 10),
+        const Divider(height: 1, color: grey2),
+        Padding(
+            padding: const EdgeInsets.all(8),
+            child: SizedBox(
+                width: double.infinity,
+                child: ElevatedButton(
```

Comment:
Could you make the button style similar to the other ones in the app? For example the button to add/edit a transaction.

Reply:
Fixed